### ISSUE TITLE
[A11y] Collection Landing - Image Logo Zoom

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1837]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

At 400% zoom and 1280x1024, the OSULP logo requires horizontal scrolling to read.

### Suggested Fix

The image is given a set height which doesn't seem to change at any zoom level.  I think the fix would just be to make its width a maximum of 100% or something, and not size it based on height.  When zoomed in a lot, the logo is way too big anyway, because 100px is huge at 400% zoom.

If this image can be turned into a purely decorative image by removing the text and making it just a simple logo (perhaps with text next to it), that would likely solve the problem as well.

**Comment by jechols** [comment]
QA: Set desktop resolution to 1280x1024, zoom the page to 400%, verify you don't need to scroll to see anything valuable (like text)

**Comment by jechols** [comment]
Unfortunately this isn't fixed - at 1280x1024 and 400%, the image still requires scrolling to see the text:

![image](https://user-images.githubusercontent.com/2157974/132393209-284ff8c6-2d15-4d1d-bb76-395b0abb8f66.png)

This is probably very low priority in terms of real-world effects, but it's also a WCAG 2.1 AA requirement: see [reflow requirements](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html). This is likely one of those issues where no user is actually going to be hurt by it, because the value of the text in that image is low, but it is *technically* still a AA failure. @CGillen, I leave it to you to decide how best to prioritize this.

**Comment by jechols** [comment]
I'm still seeing the original problem. The DOM, on initial load, shows this:

```html
<a role="button" type="button" class="collapsed collapse-toggle collapse activity-toggle" data-toggle="collapse" href="#collapseUserActivity" onclick="toggleCollapse(this)" aria-controls="collapseUserActivity"><span class="fa fa-line-chart" aria-hidden="true"></span> <span>Your activity</span></a>
```
I don't know if it's related, but the `<ul>` element actually has an `aria-expanded` attribute, and it shouldn't. It sounds like that's unrelated, but could something in code or JS be applying the attribute to the wrong element?
